src/app/research/page.tsx

```diff
@@ -1,5 +1,12 @@
 const Research = () => {
-  return <div>research comps here</div>;
+  return (
+    <div className="flex flex-col items-center justify-center">
+      <p className="p-6 font-seasons text-5xl">RESEARCH</p>
+      <p className="text-center">
+        Click on these for access to more <br /> resources!
+      </p>
+    </div>
```

Comment:
```suggestion
  return (
    <div className="">

    </div>
```
remove this this is someone else's issue, we're trying to give everyone easy issues for the first week